cmd/root.go

```diff
@@ -5,32 +5,44 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/opdev/opcap/internal/logger"
 	"github.com/spf13/cobra"
 )
 
+var logLevel string
+
 // rootCmd represents the base command when called without any subcommands
-var rootCmd = &cobra.Command{
-	Use:   "opcap",
-	Short: "A brief description of your application",
-	Long: `A longer description that spans multiple lines and likely contains
+func rootCmd() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "opcap",
+		Short: "A brief description of your application",
+		Long: `A longer description that spans multiple lines and likely contains
 examples and usage of using your application. For example:
```

Comment:
Shouldn't we have a description here?

Reply:
This is addressed in #240. If we go ahead and merge that, I'll rebase on top of it.

Reply:
Ok. Makes sense.